src/utils.c

```diff
@@ -15,47 +15,48 @@ void system_settings() {
     check_system_pointer();
     if (pointer_size == 4) {
         min_chunk_size = 16;
-        log_info("Minimum chunk size is 8 bytes (4 free).");
+        // log_info("Minimum chunk size is 8 bytes (4 free).");
     }
     else {
         min_chunk_size = 16;
-        log_info("Minimum chunk size is 16 bytes (8 free).");
+        // log_info("Minimum chunk size is 16 bytes (8 free).");
     }
 }
 
 void check_system_pointer() {
     pointer_size = sizeof(void*);
-    if (pointer_size == 4) {
-        log_info("System has 4-byte pointers.");
-    }
-    else if (pointer_size == 8) {
-        log_info("System has 8-byte pointers.");
-    }
-    else {
-        printf("Unexpected pointer size: %zu bytes\n", pointer_size);
-        // add error exit
-    }
+    // if (pointer_size == 4) {
```

Comment:
maybe we don't necessarily need this code except for debugging? it clogs up the test runs a bit :)